airflow-core/src/airflow/api_fastapi/execution_api/routes/__init__.py

```diff
@@ -48,5 +49,6 @@
 )
 authenticated_router.include_router(variables.router, prefix="/variables", tags=["Variables"])
 authenticated_router.include_router(xcoms.router, prefix="/xcoms", tags=["XComs"])
+authenticated_router.include_router(hitl.router, prefix="/hitl-responses", tags=["Human in the Loop"])
```

Comment:
I'd propose to leave the "responses" out, might be useful if some other stuff might be needed as well.
```suggestion
authenticated_router.include_router(hitl.router, prefix="/hitl", tags=["Human in the Loop"])
```

Reply:
what bugs me is that `hitl` itself is not an object or resource. `hitl-response` is probably not really better. I'll go with `hitl-task` for now

Reply:
just updated it to `hitl-details`